onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -100,9 +100,9 @@ def optimize_by_onnxruntime(
         )
     else:
         session = onnxruntime.InferenceSession(
-            onnx_model_path, sess_options, providers=["CUDAExecutionProvider"], **kwargs
+            onnx_model_path, sess_options, providers=["CUDAExecutionProvider", "ROCMExecutionProvider"], **kwargs
         )
-        assert "CUDAExecutionProvider" in session.get_providers()  # Make sure there is GPU
+        assert "CUDAExecutionProvider" or "ROCMExecutionProvider" in session.get_providers()  # Make sure there is GPU
 
     assert os.path.exists(optimized_model_path) and os.path.isfile(optimized_model_path)
     logger.debug("Save optimized model by onnxruntime to {}".format(optimized_model_path))
```

Comment:
py lint. This can be changed to `logger.debug("Save optimized model by onnxruntime to %s", optimized_model_path)` to delay the string formation.